tests/playwright/specs/wikipedia.js

```diff
@@ -0,0 +1,8 @@
+var { chromium } = require("playwright");
+
+(async () => {
```

Comment:
I wouldnt call this a test. You need to setup a test runner like jest to run the tests. I guess I wasnt explicit. Ill update the issue

Reply:
Write assertion for an element on the page
https://playwright.dev/#version=v1.6.2&path=docs%2Fassertions.md&q=

Reply:
This has been noted.